lib/splicer/dns_made_easy/record.rb

```diff
@@ -2,7 +2,10 @@ module Splicer
   module DnsMadeEasy
 
     class Record
-      attr_reader :id, :type, :value, :ttl, :data
+      attr_reader :name, :value, :id, :type, :dynamicDns,
```

Comment:
Why do I need to get rid of add_attribute?
Note: I tried removing add_attribute and things broke.